packages/jobs-service-webapp/src/index.html

```diff
@@ -0,0 +1,55 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~ 
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~ 
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.      
+-->
+
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Jobs Service Webapp</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.svg" />
+  </head>
+  <body>
+    <div class="maindiv">
+      <div class="container">
+        <div class="logo" style="display: flex; justify-content: center">
+          <img src="favicon.svg" alt="KIE Logo" />
+          <h1>Jobs Service</h1>
+        </div>
+        <div>
+          <h4>Your Jobs service is up and working!</h4>
+        </div>
+        <p>
+          The Job Service facilitates the scheduled execution of tasks in a cloud environment. These tasks are
+          implemented by independent services, and can be started by using any of the Job Service supported interaction
+          modes, based on Http calls or Knative Events delivery.
+        </p>
+        <div>
+          <a
+            href="https://kiegroup.github.io/kogito-docs/serverlessworkflow/latest/job-services/core-concepts.html"
+            class="btn"
+            target="_blank"
+            >SonataFlow Jobs Service docs</a
```

Comment:
We should try not to reference SonataFlow in this landing page.
```suggestion
            >Jobs Service docs</a
```